src/ServiceBus/ServiceBus.Autorest/test/Set-AzServiceBusSubscription.Tests.ps1

```diff
@@ -29,34 +29,10 @@ function AssertSubscriptionUpdates{
     $expectedSub.ClientId | Should -Be $actualSub.ClientId
     $expectedSub.IsShared | Should -Be $actualSub.IsShared
     $expectedSub.IsDurable | Should -Be $actualSub.IsDurable
-
-    if ($expectedSub.DefaultMessageTimeToLive -gt (New-TimeSpan -Days 10675197)){
-        $actualSub.DefaultMessageTimeToLive | Should -BeGreaterOrEqual (New-TimeSpan -Days 10675197)
-    }
-    else{
-        $actualSub.DefaultMessageTimeToLive | Should -Be $expectedSub.DefaultMessageTimeToLive
-    }
-
-    if ($expectedSub.DuplicateDetectionHistoryTimeWindow -gt (New-TimeSpan -Days 10675197)){
-        $actualSub.DuplicateDetectionHistoryTimeWindow | Should -BeGreaterOrEqual (New-TimeSpan -Days 10675197)
-    }
-    else{
-        $actualSub.DuplicateDetectionHistoryTimeWindow | Should -Be $expectedSub.DuplicateDetectionHistoryTimeWindow
-    }
-
-    if ($expectedSub.LockDuration -gt (New-TimeSpan -Days 10675197)){
-        $actualSub.LockDuration | Should -BeGreaterOrEqual (New-TimeSpan -Days 10675197)
```

Comment:
As your test cases changes, please re-record these tests cases and upload their new recording files